vrecord

```diff
@@ -612,8 +614,7 @@ _yes_or_no(){
 }
 
 _frames_to_hhmmss(){
-    FRAMERATE=$(grep "fps=" "${LOGDIR}/${ID}${CAPTURELOGSUFFIX}" | cut -d' ' -f 5)
-    NUM=$(bc <<< "scale=0; ${i} / ${FRAMERATE}")
+    NUM=$(bc <<< "scale=0; ${i} / ${DECKLINK_FPS}")
```

Comment:
maybe simpler as `NUM=$(bc <<< "${i} / (${DECKLINK_FPS})")` and then leave `DECKLINK_FPS="30000/1001"` and `DECKLINK_FPS="25000/1000"`